amqp-format.md

```diff
@@ -40,8 +40,8 @@ interpreted as described in [RFC2119][RFC2119].
 
 This section defines how CloudEvents attributes are mapped to the AMQP
 type-system. This specification does not explicitly map each attribute, but
-provides a generic mapping model that applies to all current and future Cloud
-Event attributes.
+provides a generic mapping model that applies to all current, future and
+extension, Cloud Event attributes.
```

Comment:
The commas here confuse me, especially the comma after `extension`. This change doesn't seem important to me, and I'd prefer to remove it. If it's important to others to emphasize that extensions are attributes, could we rewrite it as "[this] applies to all current and future Cloud Event attributes, including extensions"

Reply:
I like your proposed text much better - thanks! I wasn't thrilled with the wording I had.